closed/autoconf/custom-hook.m4

```diff
@@ -360,25 +370,37 @@ AC_DEFUN([OPENJ9_PLATFORM_SETUP],
       OPENJ9_PLATFORM_CODE=xa64
     elif test "x$OPENJDK_BUILD_OS" = xwindows ; then
       OPENJ9_PLATFORM_CODE=wa64
-      if test "x$OPENJ9_LIBS_SUBDIR" = xdefault ; then
-        OPENJ9_BUILDSPEC=win_x86-64
+      if test "x$OPENJ9_ENABLE_MIXED_REFERENCES" = xfalse ; then
+        if test "x$OPENJ9_LIBS_SUBDIR" = xdefault ; then
+          OPENJ9_BUILDSPEC=win_x86-64
+        else
+          OPENJ9_BUILDSPEC=win_x86-64_cmprssptrs
+        fi
```

Comment:
Rather than repeating tests for the three modes, all we really need to do is change the prefix of `OPENJ9_BUILDSPEC` from 'windows' to 'win'; likewise for macosx (removing 'mac').